hooks/useChat.tsx

```diff
@@ -40,7 +44,7 @@ const useChat = () => {
     },
     body: {
       accountId,
-      address,
+      address: address ? address : undefined,
```

Comment:
This is the same functionality with less code, right?
```suggestion
      address
```